app/src/test/java/com/x8bit/bitwarden/ui/auth/feature/accountsetup/SetupUnlockViewModelTest.kt

```diff
@@ -0,0 +1,98 @@
+package com.x8bit.bitwarden.ui.auth.feature.accountsetup
+
+import androidx.lifecycle.SavedStateHandle
+import app.cash.turbine.test
+import com.x8bit.bitwarden.data.auth.repository.AuthRepository
+import com.x8bit.bitwarden.data.auth.repository.model.UserState
+import com.x8bit.bitwarden.data.platform.manager.BiometricsEncryptionManager
+import com.x8bit.bitwarden.data.platform.repository.SettingsRepository
+import com.x8bit.bitwarden.data.platform.repository.model.Environment
+import com.x8bit.bitwarden.ui.platform.base.BaseViewModelTest
+import io.mockk.every
+import io.mockk.mockk
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import javax.crypto.Cipher
+
+class SetupUnlockViewModelTest : BaseViewModelTest() {
```

Comment:
Are we omitting the tests for the other actions cause they are similar to the AccountSecurityViewModel test?

Reply:
They just don't do anything yet, they are just empty TODOs.

I am working on them as part of my next ticket

Reply:
ope! don't mind me